nss-cohort-37/Workforce-Silver-Snakes
# removed edit from computers and fix conditional delete on details

# Description
REMOVED edit from computers and ADDED conditional delete on details
## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. View all Computers and verify no edit button on table rows.
2. Also check for delete where employees are not present on computer.

# Checklist:
